juniper_official/System/db-size.yml

```diff
@@ -4,10 +4,10 @@ DBSizeTable:
   item: .
   key:
   view: DBSizeView
- 
+
 DBSizeView:
   fields:
-    maxsize: maximum-size
-    cursize: current-size
-    actsize: actual-usage
-    avaisize: available-space
+    maxsize: {maximum-size: Search=(\d+\.\d+)}
+    cursize: {current-size: Search=(\d+\.\d+)}
+    actsize: {actual-usage: Search=(\d+)\.\d+}
```

Comment:
bracket"(" closing is not in proper.
correct one is below
actsize: {actual-usage: Search=(\d+\.\d+)}